catalog-rest-service/src/main/java/org/openmetadata/catalog/security/SecurityUtil.java

```diff
@@ -55,6 +56,20 @@ public static void checkAdminRoleOrPermissions(
     }
   }
 
+  public static void checkAdminRoleOrPermissions(
```

Comment:
@harshach 

This is the overloaded function that does checkAdmin, role and metadata operation. 
Going to attempt hooking this up to the resource APIs.